src/Compilation.zig

```diff
@@ -1428,6 +1437,18 @@ pub fn create(gpa: Allocator, options: InitOptions) !*Compilation {
             system_libs.putAssumeCapacity(lib_name, options.system_lib_infos[i]);
         }
 
+        var lib_dirs = std.ArrayList([]const u8).init(arena);
+        try lib_dirs.appendSlice(options.lib_dirs);
+        try lib_dirs.appendSlice(libc_dirs.libc_lib_dir_list);
+
+        var framework_dirs = std.ArrayList([]const u8).init(arena);
+        try framework_dirs.appendSlice(options.framework_dirs);
+        try framework_dirs.appendSlice(libc_dirs.libc_framework_dir_list);
+
+        var rpath_list = std.ArrayList([]const u8).init(arena);
+        try rpath_list.appendSlice(options.rpath_list);
+        try rpath_list.appendSlice(libc_dirs.libc_rpath_list);
```

Comment:
I don't think these should be added here. This information is available to whatever logic needs it via `link.File.options.libc_installation`

Reply:
Hmm, all are needed by the linker, and framework paths are needed by the frontend (compiler) too. How else would you provide this info then?

Reply:
Both the linker and the frontend have access to `link.File.options.libc_installation`.

Reply:
Oh, did I finally understand it that you mean we can recreate this info on request in whatever linker needs it?

Reply:
yes I believe that is correct :+1:

Reply:
Took me long enough to get it...